### ISSUE TITLE
[COGNITION]: Primary Button - CONSIDER adding screen reader text and icon for primary calls to action

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10477]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The [Primary Button derivative](https://design.va.gov/components/buttons#primary-buttons) of our button component has an interesting note in it from user research:

> Our researchers learned that users were missing calls to action on some pages because the blue buttons did not stand out, so a green primary button was created, tested and determined to be successful.<br/>
[Research Insight](https://design.va.gov/components/buttons#research-insight)

I'd like to explore adding an SR-only span or descriptive text in cases where we use this green button for primary calls to action. Is there a way we could add aural (voice) emphasis and an icon or other visual treatment beyond color to show that this is primary action?

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Button includes an `aria-label` or (ideally) a `<span class="usa-sr-only">` inline element that announces the visual text and screen reader emphasis
- [ ] Button includes an icon or cue that doesn't rely on color to show it's a primary action
- [ ] Zero [axe plugin](https://deque.com/axe) violations

## Environment

* https://design.va.gov/components/buttons#primary-buttons

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

<img width="817" alt="Screen Shot 2020-06-19 at 11 51 02 AM" src="https://user-images.githubusercontent.com/934879/85159468-2b672180-b223-11ea-825c-885a7fb8a8ab.png">

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #212](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/212) via [**ZenHub**](https://www.zenhub.com/)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #213](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/213) via [**ZenHub**](https://www.zenhub.com/)
